frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -485,13 +469,11 @@ pub mod pallet {
 					);
 				}
 				Self::deposit_event(Event::TokenLockedConfirmed(
-					message_id, user, amount, result,
+					*lane, nonce, user, amount, result,
 				));
-				weight = weight.saturating_add(
-					<T as frame_system::Config>::DbWeight::get().reads_writes(2, 3),
-				);
 			}
-			weight
+			// TODO: The returned weight should be more accurately. See: https://github.com/darwinia-network/darwinia-common/issues/911
+			<T as frame_system::Config>::DbWeight::get().reads_writes(1, 1)
```

Comment:
This weight is out-of-data. It should be reads = 1, writes = 0;

Reply:
where's the update?

Reply:
We only read once in `let (user, amount) = <TransactionInfos<T>>::take(message_id);` right? then, no other storage operation.

Reply:
the deposit event will write, and the transfer will also write storage.